### PR TITLE
[TINKERPOP-2820] Fixed connection closing for sessioned client in gremlin-python

### PR DESCRIPTION
**Pull request by vkagamlyk**
Python Client did not close the connection when using session.

https://issues.apache.org/jira/browse/TINKERPOP-2820

### CODE REVIEW

**Comment by xiazcy** [comment]
LGTM VOTE + 1

File: gremlin-python/src/main/python/gremlin_python/process/graph_traversal.py [hunk 1]
```diff
@@ -1092,6 +1092,7 @@ def __verify_transaction_state(self, state, error_message):
     def __close_session(self, session):
         self._session_based_connection.close()
```
Comment (kenhuuu) [inline gremlin-python/src/main/python/gremlin_python/process/graph_traversal.py:1093]: Would it make sense to put these two lines in the newly added remove_session()?

**Comment by kenhuuu** [comment]
VOTE +1

**Comment by vkagamlyk** [comment]
VOTE +1

**Pull request merged**